stories/ToggleRefinement.stories.js

```diff
@@ -20,6 +20,15 @@ storiesOf('ais-toggle-refinement', module)
       />
     `,
   }))
+  .add('with an on value (with multiple values)', () => ({
+    template: `
+      <ais-toggle-refinement
+        attribute="brand"
+        label="Brands starting with \"i\""
```

Comment:
```suggestion
        label="Metra or Samsung"
```

Reply:
Oh my bad, I totally forgot about this label.

Thanks!